.gitignore

```diff
@@ -13,6 +13,8 @@ bin
 *.out
 coverage.txt
 
+.vscode
```

Comment:
We don't plan to include any IDE-specific entry here, note we didn't include the common `.idea` for JetBrains IDEs neither.

Reply:
Was just trying to keep myself from accidentally committing any of this. Will remove it.

Reply:
In such case, I'd suggest you to have your own `.gitconfig`, and set it up globally.